lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml

```diff
@@ -430,6 +562,19 @@ states:
         targetJourney: FAILED
         targetState: FAILED_CONFIRM_DETAILS
 
+  STRATEGIC_APP_TRIAGE:
+    nestedJourney: STRATEGIC_APP_TRIAGE
+    exitEvents:
+      next:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+      end:
+        targetJourney: FAILED
+        targetState: FAILED_UPDATE_DETAILS
+      multipleDocCheckPage:
+        targetJourney: TECHNICAL_ERROR
+        targetState: ERROR
+
```

Comment:
Hmm so there's a bunch of different end states for these CoI scenarios which we'll need to handle on the handback from the app (when we get to it)